rcsb/db/processors/ClusterDataPrep.py

```diff
@@ -61,7 +62,8 @@ def __init__(self, **kwargs):
         self.__entityAttributeName = kwargs.get("entityAttributeName", "entity_id")
         self.__instanceAttributeName = kwargs.get("instanceAttributeName", "instance_id")
         # Just a note for the assumned naming conventions clusters-by-%(clusterType)s-%(level)s.txt of the cluster data files
-        self.__clusterFileNameTemplate = kwargs.get("clusterFileNameTemplate ", "clusters-by-%(clusterType)s-%(level)s.txt")
+        clusterFileNameTemplate = kwargs.get("clusterFileNameTemplate", None)
+        self.__clusterFileNameTemplate = clusterFileNameTemplate if clusterFileNameTemplate else "clusters-by-%(clusterType)s-%(level)s.txt"
```

Comment:
I'm curious why to choose this syntax vs the old one? The old one seems more readable. But perhaps I'm missing something?

Reply:
I had to do it this way because I set the default value of `clusterFileNameTemplate=None` in the method calls between Luigi and this point, as it didn't seem good practice to explicitly set the default value over and over again. So, this is the only spot where the default value of `"clusters-by-%(clusterType)s-%(level)s.txt"` will be assigned to `clusterFileNameTemplate` apart from the `etl_config.py` in Luigi.